### PR TITLE
Add dataloader for portrait segmentation

### PR DESCRIPTION
**Pull request by seldauyanik-maxim**
Data loader module for AISegment: a portrait segmentation dataset is
added with several variants using high and low resolution images and
using disk or memory for data storage.

Training and evaluation scripts are added for training and evaluating
portrait segmentation dataset using high resolution UNET model

### CODE REVIEW

**Review by rotx-maxim — CHANGES_REQUESTED** [review]
See https://github.com/seldauyanik-maxim/ai8x-training/pull/1

**Review by ermanok — APPROVED** [review]
I only have a minor comment, rest looks good.

**Review by MaximGorkem — APPROVED** [review]
Looks good

File: scripts/evaluate_aisegment_unet.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+python3 train.py --data /data2/ml/ --deterministic --model ai85unetlarge --dataset AISegment_352 --device MAX78000 --qat-policy qat_policy_aisegment.yaml --use-bias --evaluate -8 --exp-load-weights-from ../ai8x-synthesis/trained/ai85-aisegment-unet-large-q.pth.tar "$@"
```
[the remark this answers is not in the document]
Comment (seldauyanik-maxim) [reply scripts/evaluate_aisegment_unet.sh]: First commit will be amended to add this option

**Review by rotx-maxim — APPROVED** [review]

**Pull request merged**